src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { Person } from '../../types/Person';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+
+  useEffect(() => {
+    (async () => {
+      try {
+        setIsLoading(true);
+        const peopleFromServer = await getPeople();
+
+        setPeople(peopleFromServer);
+      } catch {
+        setHasError(true);
+      } finally {
+        setIsLoading(false);
+      }
+    })();
+  }, []);
+
+  return (
+    <>
+      <div className="container">
+        <h1 className="title">People Page</h1>
+        <div className="block">
+          <div className="box table-container">
+            {isLoading
+              ? <Loader />
+              : (
+                <>
+                  {hasError && (
+                    <p data-cy="peopleLoadingError" className="has-text-danger">
+                      Something went wrong
+                    </p>
+                  )}
+
+                  {people.length <= 0 && (
+                    <p data-cy="noPeopleMessage">
+                      There are no people on the server
+                    </p>
+                  )}
+
+                  <PeopleTable people={people} />
```

Comment:
Let's render it only if there are no errors
![Screenshot 2023-04-18 at 19 21 52](https://user-images.githubusercontent.com/84275496/232841483-cb1e6fbb-5895-4727-87a4-913d3900a34d.png)